src/core/reference/include/openvino/reference/string_tensor_pack.hpp

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+#include "openvino/core/shape.hpp"
+
+namespace ov {
+namespace reference {
+template <typename T_idx>
+void string_tensor_pack(const T_idx* begins,
+                        const T_idx* ends,
+                        const uint8_t* symbols,
```

Comment:
Consider to change it to char* and data from tensor can reinterpreted to use this type.
or re interpret this pointer in reference implementation.

Reply:
Done.